### DISCUSSION TITLE
Book and multi-page document viewer

### DISCUSSION THREAD

**Discussion by stothert** [MukurtuCMS/Mukurtu-CMS#24] [Mukurtu 4 Core Features for Launch]
My organization has many multipage items that could be much more user-friendly and load faster with the addition of viewer similar to the Internet Archive Book Reader: https://archive.org/details/BookReader
The Universal Viewer: https://universalviewer.io/
Or simply a side by side PDF flipbook similar to commercial products such as issu: https://issuu.com/

I have a dream of virtually replicating the experience of sitting in an archives with a single folder of archival material in front of you. You open the folder and you see the front and back (even if blank) of each item. My favorite viewer is the Internet Archive Reader because it includes the virtual page turns. Rather than handle each letter as a it's own compound object with item level metadata, you could leverage the level of description in a finding aid and provide access to that folder level material as a group.

For many archives, future digitization projects need to be more "mass" and "batch" rather than "item level" and "boutique". The link to PDFs in Mukurtu 3 leaves much to be desired. It would be lovely to have lots of contents in a contained and attractive viewer that is easy to describe in a single digital heritage record.

It would also be useful to control display and not rely on browser discrepancies in how PDFs are handled.

Thanks!

**Comment by alexmerrill** [comment]
The multipage document experience is also on our list for improvement. The tools you mention are good examples of the goalposts we are aiming for user experience.  One of the wrinkles we have been talking about in relation to tools/viewers like this is the continued ability to create metadata and/or community records at the page level and how a viewer like this would impact displaying that metadata/ community record at the point at which it was created and in context.  

For your use case would how important is the continued ability to see page-level metadata and/or community records in the viewer or would a separate view be sufficient (instead of the link to PDF there was a book reader link to viewer instead).  

These viewers also have some sort of technical overhead.  The Internet Archive is one of the simpler ones, I think, mostly written in some sort of javascript and pulling from a directory or collection of items but some, like the Universal Viewer, needs a little more,  it looks to be mostly written as an IIIF OpenSeaDragon type viewer which would need an IIF standard manifest be produced from somewhere.  

In your use case would how important is it to keep the technical debt to a minimum or would there be capacity outside of a standard Mukurtu CMS installation for folks at your institution.

Thanks for the input and I hope to hear more about your thoughts based on your use cases!

Alex
